[Build Failure]: Missing `afxwin.h` when building with `-D BUILD_EDITOR=ON`

### Build Version

v1.5 ae626b36eb6ad6f8ff3004cf4c23040406bf9c1f

### Operating System Environment

- [ ] Microsoft Windows (32-bit)
- [X] Microsoft Windows (64-bit)
- [ ] Mac OS X
- [ ] Linux (specify distribution and version below)

### CPU Environment

- [X] x86 (32-bit Intel/AMD)
- [ ] x86\_64 (64-bit Intel/AMD)
- [ ] ARM (32-bit)
- [ ] ARM64 (64-bit; sometimes called AArch64)
- [ ] Other (RISC V, PPC...)

### Description

If I try to build Descent 3 with `-D BUILD_EDITOR=ON`, then it fails with this error:

```
D:\Descent 3 build test\Descent3\editor\stdafx.h(31): fatal error C1083: Cannot open include file: 'afxwin.h': No such file or directory
```

### Regression Status

I don’t know.

### Steps to Reproduce

1. Uninstall the Visual Studio Installer. This will help make sure that you don’t have any extra Visual Studio components installed that might prevent this bug from occurring.
2. Follow [Descent 3’s Windows build instructions](https://github.com/DescentDevelopers/Descent3/blob/1ed931ff60faa3abd28bdac53b25a1ff5d1e839e/README.md#building---windows). When performing the instructions, be sure to do the following:

	- Run the two commands that the instructions provide in order to install Visual Studio 2022 with the “Desktop development with C++” workload. This will also help make sure that you don’t have any extra Visual Studio components installed that might prevent this bug from occurring.
	- When you get to the part where it tells you to run the first `cmake` command, remember to add `-D BUILD_EDITOR=ON` to the end of the command-line.

--- a/README.md
+++ b/README.md
@@ -93,7 +93,7 @@ The milestone needs testing on all platforms. Please report issues when found.
 
 ## Building
 #### Building - Windows
-1. Make sure that you have Git and Visual Studio 2022 with the “Desktop development with C++” workload. If you don’t already have those installed or you aren’t sure, then open an elevated Command Prompt and run:
+1. Make sure that you have Git and Visual Studio 2022 with the “Desktop development with C++” workload and the “C++ MFC for latest v143 build tools (x86 & x64)” component. If you don’t already have those installed or you aren’t sure, then open an elevated Command Prompt and run:
 
     <!--
     The following code block specifies the full path to the Visual Studio Installer because the Visual Studio Installer doesn’t add itself to the user’s Path. The installer is guaranteed to be in a specific location on 64-bit systems [1]. The installer will be in a different location on 32-bit systems [2], but Visual Studio 2022 doesn’t support 32-bit systems [3] so we can ignore that detail.
@@ -110,7 +110,8 @@ The milestone needs testing on all platforms. Please report issues when found.
         --passive^
         --channelId VisualStudio.17.Release^
         --productId Microsoft.VisualStudio.Product.Community^
-        --add Microsoft.VisualStudio.Workload.NativeDesktop;includeRecommended
+        --add Microsoft.VisualStudio.Workload.NativeDesktop;includeRecommended^
+        --add Microsoft.VisualStudio.Component.VC.ATLMFC
     ```
 
 2. Open a “x86 Native Tools Command Prompt” and run:
@@ -118,7 +119,7 @@ The milestone needs testing on all platforms. Please report issues when found.
     ```batch
     git clone https://github.com/DescentDevelopers/Descent3
     cd Descent3
-    cmake --preset win32 -D ENABLE_LOGGER=[ON|OFF]
+    cmake --preset win32 -D ENABLE_LOGGER=[ON|OFF] -D BUILD_EDITOR=[ON|OFF]
     cmake --build --preset win32 --config [Debug|Release]
     ```
 

